Makefile

```diff
@@ -0,0 +1,44 @@
+XCARGS := -workspace VelocidiSDK.xcworkspace \
+		-sdk "iphonesimulator" \
+    -destination "platform=iOS Simulator,OS=14.4,name=iPhone 11 Pro Max" \
+    GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=YES GCC_GENERATE_TEST_COVERAGE_FILES=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+
+all: clean build
+
+clean:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK  clean | xcpretty
+
+build:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK build | xcpretty
+
+test:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme VelocidiSDK test | xcpretty
+
+examples: install-examples build-objc-example build-swift-example
+
+install-examples:
+	set -o pipefail && pod install --project-directory=Examples/
+
+build-objc-example:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme ObjcExample clean build | xcpretty
+
+build-swift-example:
+	set -o pipefail && xcodebuild $(XCARGS) -scheme SwiftExample clean build | xcpretty
+
+install:
+	pod install
+
+prerequisites:
+	.scripts/prerequisites.sh
+
+oclint:
+	set -o pipefail && \
+	xcodebuild -scheme VelocidiSDK -sdk iphoneos -workspace VelocidiSDK.xcworkspace clean && \
```

Comment:
Can't we save a line here and do `xcodebuild -scheme VelocidiSDK -sdk iphoneos -workspace VelocidiSDK.xcworkspace COMPILER_INDEX_STORE_ENABLE=NO clean build` instead?

Reply:
Fixed in 6aaf770